scripts/ubuntu/1-install-dependencies.sh

```diff
@@ -54,18 +36,25 @@ sudo DEBIAN_FRONTEND=noninteractive apt-get upgrade -y
 
 # TODO Comment on what dependency is required for:
 packagelist=(
-    "docker-ce (>=5:24.0.7-1~ubuntu.22.04~jammy)"    # Test Harness uses Docker
-    "python3-pip (=22.0.2+dfsg-1ubuntu0.4)"          # Test Harness CLI uses Python              
-    "python3-venv (=3.10.6-1~22.04)"                 # Test Harness CLI uses Python
+    "docker-ce (>=5:24.0.7-1~ubuntu.22.04~jammy)"     # Test Harness uses Docker
+    "python3-pip (>=22.0.2+dfsg-1ubuntu0.4)"          # Test Harness CLI uses Python              
+    "python3-venv (>=3.10.6-1~22.04)"                 # Test Harness CLI uses Python
 )
 
+UBUNTU_VERSION_NUMBER=$(lsb_release -sr)
+if [UBUNTU_VERSION_NUMBER -eq "22.04"]; then
+  packagelist+="linux-modules-extra-raspi (>=5.15.0.1046.44)"
+fi
+
```

Comment:
I noticed that this package used to be installed on the script for raspi dependencies. Is this necessary for a non-raspi setup as well? Or could this be moved to that script?